themes/theme-b2b/src/components/DateField.styles.ts

```diff
@@ -10,9 +10,10 @@ export const DateField: ThemeComponent<'DateField'> = {
     'h-[34px]',
     'data-[hovered]:border-border-hover',
     // inputDisabled('data-[disabled]') color tokens are not working with in this component
-    'data-[disabled]:bg-bg-disabled data-[disabled]:border-border-disabled data-[disabled]:text-text-disabled data-[disabled]::cursor-not-allowed',
+    'data-[disabled]:bg-bg-disabled data-[disabled]:border-border-disabled data-[disabled]:text-text-disabled data-[disabled]:cursor-not-allowed',
     // inputFocus('data-[focus-within]') is not working with this component
```

Comment:
I don't understand the comment here. Does that mean the style doesn't work? If so, why do we have it in there?

Reply:
Oh that comments are there, because for reasons the `input` styles where not working. You know ?